actor.go

```diff
@@ -28,89 +25,106 @@ type ActorConfig struct {
 // NewActor returns a new Actor
 func NewActor(c ActorConfig) (*Actor, error) {
 	if c.MailboxSize < 1 {
-		return nil, MailboxSizeTooSmallError("mailbox must be greater than 0")
+		return nil, ActorConfigurationError("mailbox must be greater than 0")
```

Comment:
this whole error could be static, right? just because you only use it in one place right now